### PR TITLE
Clarify errormsg in ImageModel.scala

### PR DESCRIPTION
**Pull request by cyita**
If the input is empty, throw an error message before calling model.predictImage.

### CODE REVIEW

**Comment by qiuxin2012** [comment]
fixed #1483

**Comment by qiuxin2012** [comment]
Add a unit test `should throw exception`

File: zoo/src/test/scala/com/intel/analytics/zoo/models/image/imageclassification/ImageClassificationSpec.scala [hunk 1]
```diff
@@ -138,22 +136,50 @@ class ImageClassificationSpec extends ZooSpecHelper {
...
   }
 
   "ImageClassifier" should "predict bigdl inception-v1-quantize" in {
     predict("https://s3-ap-southeast-1.amazonaws.com/bigdl-models/imageclassification/imagenet/" +
-      "bigdl_inception-v1-quantize_imagenet_0.4.0.model", "bigdl")
+      "bigdl_inception-v1-quantize_imagenet_0.4.0.model", "bigdl", resource.getFile)
+  }
+
+  "ImageClassifier" should "throw exception if input is empty locally" in {
+    val path = "./tempEmpty"
```
Comment (qiuxin2012) [inline zoo/src/test/scala/com/intel/analytics/zoo/models/image/imageclassification/ImageClassificationSpec.scala]: It's not save to use a fixed directory, please use `com.google.common.io.Files.createTempDir()`

**Comment by qiuxin2012** [comment]
is jenkins passed?

**Comment by cyita** [comment]
> is jenkins passed?

passed

**Pull request merged**